### PR TITLE
[daheinze-oscheer] added pointers and errors

### LINKED ISSUE #25: [daheinze-oscheer] Pointers & errors
Complete the Pointers & errors lesson

### PR DESCRIPTION
**Pull request by oliverscheer**
added pointers and errors. 

what is missing in the tutorial is the fact that you need to install errcheck and after that add it to the path
export GOPATH="$HOME/go"
PATH="$GOPATH/bin:$PATH"

### CODE REVIEW

File: daheinze-oscheer/go.sum [hunk 1]
```diff
@@ -0,0 +1,31 @@
+github.com/kisielk/errcheck v1.4.0 h1:ueN6QYA+c7eDQo7ebpNdYR8mUJZThiGz9PEoJEMGPzA=
```
Comment (starlord-daniel) [inline daheinze-oscheer/go.sum:1]: You don't have to push this file

File: daheinze-oscheer/pointersanderrors/wallet.go [hunk 2]
```diff
@@ -0,0 +1,46 @@
...
+
+import (
+	"errors"
+	"fmt"
+)
+
+// Stringer lorem ipsum
+type Stringer interface {
+	String() string
+}
```
Comment (starlord-daniel) [inline daheinze-oscheer/pointersanderrors/wallet.go]: Do we still need this?

**Review by starlord-daniel — APPROVED** [review]

File: daheinze-oscheer/pointersanderrors/wallet.go (hunk 2 above)
Comment (oliverscheer) [reply daheinze-oscheer/pointersanderrors/wallet.go]: not really, as no other "class" is using this, it is not necessary to define an interface here. I removed it.

File: daheinze-oscheer/go.sum (hunk 1 above)
Comment (oliverscheer) [reply daheinze-oscheer/go.sum:1]: you are right. but it was installed based on the tutorial. i remove it.

**Pull request merged**